code/game/objects/structures/razorwire.dm

```diff
@@ -17,6 +17,13 @@
 	max_integrity = RAZORWIRE_MAX_HEALTH
 	var/soak = 5
 
+/obj/structure/razorwire/foam
+	desc = "A bundle of barbed wire supported by metal rods, crudely built by iron-consuming nanites. Weld them to bring them to their full strength."
+	max_integrity = RAZORWIRE_MAX_HEALTH
+	obj_integrity = RAZORWIRE_MAX_HEALTH/2
+
+
+
```

Comment:
```suggestion
/obj/structure/razorwire/foam
	desc = "A bundle of barbed wire supported by metal rods, crudely built by iron-consuming nanites. Weld them to bring them to their full strength."
	max_integrity = RAZORWIRE_MAX_HEALTH
	obj_integrity = RAZORWIRE_MAX_HEALTH/2

```

Reply:
I left a bunch of spaces in, didn't I.

Reply:
Do you happen to know why that broke the PR? I can't think of any reason offhand.